things/api/things/http/endpoint.go

```diff
@@ -151,7 +151,7 @@ func listThingsEndpoint(svc things.Service) endpoint.Endpoint {
 			return nil, err
 		}
 
-		page, err := svc.ListThings(ctx, req.token, req.offset, req.limit, req.name, req.metadata)
+		page, err := svc.ListThings(ctx, req.token, req.pageMeta)
```

Comment:
I do not like the fact that previously it was called `pageMetadata`. I think  we should stay consistent. I would advise putting `pageMetadata` here as well.